### DISCUSSION TITLE
Support IPv6 for Repository Cloning

### DISCUSSION THREAD

**Discussion by robd003** [community/community#146780] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm using servers that only have IPv6 addresses and do not have any IPv4 connectivity.

At the moment I cannot git clone via either SSH or HTTPS to any GitHub repository.

Other Git hosting services offer IPv6 connectivity. I don't want to move away from GitHub, but if you don't support IPv6 then I will have to move my entire company off of GitHub.

**Comment by ScriptAutomate** [comment]
I've opened a related feature request / discussion:

- [Feature Request: Provide a centralized resource and GitHub Public Roadmap tracking of IPv6-support for GitHub Services #151477](https://github.com/orgs/community/discussions/151477)
